lighthouse-core/gather/driver.js

```diff
@@ -849,6 +849,24 @@ class Driver {
     });
   }
 
+  /**
+   * @param {number} [timeout]
+   * @return {Promise<LH.Crdp.Security.SecurityStateChangedEvent>}
+   */
+  getSecurityState(timeout = 1000) {
+    return new Promise((resolve, reject) => {
+      const err = new LHError(LHError.errors.SECURITY_STATE_TIMEOUT);
+      const asyncTimeout = setTimeout((_ => reject(err)), timeout);
+
+      this.sendCommand('Security.enable');
```

Comment:
also, @paulirish is Security domain still broken on Android, is this going to break us on real devices again?

Reply:
they fixed android like a year ago! yay

![image](https://user-images.githubusercontent.com/39191/47057937-b737c900-d177-11e8-9515-952bee4b4bf1.png)

Reply:
well someone's behind the times 😆